.github/workflows/release.yaml

```diff
@@ -13,7 +13,9 @@ on:
 
   push:
     tags:
-     - '*.*.*-*-rc'
+    ## The tag should follow the convention: VERSION - DOCKER_BUILD_COUNTER - RELEASE_TYPE
+    ## Example tags: "9.1.1-0-rc" or "9.1.1-1-alpha"
+    - '*.*.*-*-*'
```

Comment:
Should we use proper semver versions instead? This would mean `9.1.1-alpha.1` for the first alpha version of `9.1.1`. It feels weird to me to have an alpha release of a patch version though. Adding support for PLT would be at least a minor version bump in my opinion?

Whats the history on the versioning here? Is `9.1` the first PLT version?